applications/vanilla/settings/stub.php

```diff
@@ -6,19 +6,12 @@
  * @package Vanilla
  */
 
-$SQL = Gdn::Database()->SQL();
-
 // Only do this once, ever.
-$Row = $SQL->Get('Discussion', '', 'asc', 1)->FirstRow(DATASET_TYPE_ARRAY);
-if ($Row) {
-   return;
-}
-
-$DiscussionModel = new DiscussionModel();
+if (!Gdn::SQL()->Get('Discussion', '', 'asc', 1)->FirstRow(DATASET_TYPE_ARRAY)) {
```

Comment:
Giant honking if statement is bad. Either keep the early bailout above or refactor to a function.